### DISCUSSION TITLE
Landscape daytime illumination

### DISCUSSION THREAD

**Discussion by fzs1933** [Stellarium/stellarium#1608] [Q&A] [answered]
Hi,

Sorry but I have one more question:
is it possible to switch off landscape illumination at daytime?
The "show illumination" checkbox is ineffective.
Thanks

**Comment by alex-w** [comment, marked as answer]
How to get landscape illumination at daytime?

**Reply by fzs1933** [reply]
I would like to see the landscape at daytime as night - dark, only shapes.

![ill](https://user-images.githubusercontent.com/81897104/114690753-4d341680-9d17-11eb-99e5-e127dad066c0.jpg)

**Comment by ak-team** [comment]
Switch off atmosphere from the lower control bar which appears on the bottom of the screen and you will get exactly what you want ie. see the night sky during day ;) Cheers

**Reply by gzotti** [reply]
"Show illumination", as the Guide surely tells you, switches the artificial ilumination layer. 

A black landscape mask is not what made this program popular. If you really need this, create a black landscape image.

**Reply by fzs1933** [reply]
I see
ok
thank all answers
